stable/external-dns/templates/_helpers.tpl

```diff
@@ -24,28 +24,201 @@ If release name contains chart name it will be used as a full name.
 {{- end -}}
 {{- end -}}
 
-{{/* Generate basic labels */}}
-{{- define "external-dns.labels" }}
-app: {{ template "external-dns.name" . }}
-heritage: {{.Release.Service }}
-release: {{.Release.Name }}
-{{- if .Values.podLabels }}
```

Comment:
@juan131 this PR introduced a regression I believe. The pod labels are no longer being set. This is causing the combination of `aad-pod-identity` + `external-dns` to fail in our setup since `aad-pod-identity` uses labels to figure out which pods should be assigned an identity